environment_py2_linux.yml

```diff
@@ -1,5 +1,6 @@
-name: py2_parcels
+name: parcels
```

Comment:
I don't understand why all the environments have the same name.
We would like to have different names such that we can load what we want, no?